codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -805,6 +795,37 @@ private InterfaceHttpData findMultipartDisposition() {
         }
     }
 
+    private static final String FILENAME_ENCODED = HttpHeaderValues.FILENAME.toString() + '*';
+
+    private Attribute getContentDispositionAttribute(String... values) {
+        String name = cleanString(values[0]);
+        String value = values[1];
+
+        // See http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html
+        if (HttpHeaderValues.FILENAME.contentEquals(name)) {
+            // filename value is quoted string so strip them
+            int last = value.length() - 1;
+            if (value.charAt(0) == HttpConstants.DOUBLE_QUOTE && value.charAt(last) == HttpConstants.DOUBLE_QUOTE) {
```

Comment:
```if (last > 0 && ...```

Reply:
Ah yes, thank you
```
value         = token / quoted-string
```
https://tools.ietf.org/html/rfc5987#section-3.2.1